.github/workflows/test-multiple-versions.yml

```diff
@@ -47,6 +47,8 @@ jobs:
         mode: [NORMAL, PROVIDER_LESS, VERSIONED_WRITE]
         exclude:
           - { react: 16.8.6, mode: VERSIONED_WRITE }
+          - { react: 16.9.0, mode: VERSIONED_WRITE }
+          - { react: 17.0.0, mode: VERSIONED_WRITE }
```

Comment:
The new implementation depends on the new useReducer behavior in React 18, so pre-18 versions are no longer supported.